cmd/pmm-managed-starlark/main.go

```diff
@@ -46,4 +52,25 @@ func main() {
 	if on, _ := strconv.ParseBool(os.Getenv("PMM_TRACE")); on {
 		logrus.SetLevel(logrus.TraceLevel)
 	}
+
+	gobDecoder := gob.NewDecoder(os.Stdin)
```

Comment:
JSON, not Gob. I think result type has `json` tags already. If not, please add them

Reply:
https://github.com/percona-platform/platform/blob/032c4cfdf444fac4fddaa95a1e4ca806ce496ea1/pkg/check/result.go#L51-L57 – please add `json` struct tags there

Reply:
done

Reply:
@palash25 Not a typo – it is in the platform repo

Reply:
https://github.com/percona-platform/platform/pull/127

Reply:
Merged, saas repo update, please use JSON there